code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -852,3 +865,16 @@
 	if(prob(10))
 		M.say(pick("I hate my wife.", "I just want to grill for God's sake.", "I wish I could just go on my lawnmower and cut the grass.", "Yep, Quake. That was a good game...", "Yeah, my PDA has wi-fi. A wife I hate."), forced = /datum/reagent/consumable/char)
 	..()
+
+/datum/reagent/consumable/nutriment/cloth
+	name = "Cloth"
+	description = "The finest fabric in the universe..."
+	reagent_state = SOLID
+	color = "#c2bbb7"
+	chem_flags = CHEMICAL_RNG_GENERAL | CHEMICAL_RNG_FUN | CHEMICAL_GOAL_BOTANIST_HARVEST
+	taste_description = "a roll of gauze"
+	metabolization_rate = 4 * REAGENTS_METABOLISM //speedy metabolization (per tick)
+
+/datum/reagent/consumable/nutriment/cloth/on_mob_metabolize(mob/living/carbon/M)
+	holder.add_reagent(/datum/reagent/consumable/nutriment, 0.7)
+	holder.add_reagent(/datum/reagent/consumable/maltodextrin/microplastics, 0.3)
```

Comment:
```suggestion
	metabolization_rate = 2 * REAGENTS_METABOLISM //speedy metabolization (per tick)

/datum/reagent/consumable/nutriment/cloth/on_mob_metabolize(mob/living/carbon/M)
	holder.add_reagent(/datum/reagent/consumable/nutriment, 1)
	holder.add_reagent(/datum/reagent/consumable/maltodextrin/microplastics, 1)
```

This still needs to be at least 50/50 in order for the microplastics to be as effective as they need to be. 

Not even accounting for the fact I just realized there was a math error in the maltodextrin just now, but that's a problem for another PR later on.

Reply:
If you are taking a bite of cloth and 50% of it is plastic, something is seriously wrong with your diet

Reply:
Some clothes are 100% plastic. Polyester to be exact.